Automattic/ghactivity
# Add [average_label_time] shortcode & scheduled data collection

- Added new `[ghactivity_average_label_time]` shortcode which will display an average amount of time issues are remaining marked with  `[Status] Needs Review`.
- Added a scheduled task to collect that average amount of time which is running daily 
- Create a React-based component for displaying Line Chart with that data

Сonstraints:
- Currently, it is not possible to use brackets (`[`, `]`) in shortcode parameters. So these should be replaced with `&#91;` for `[` & `&#93;` for `]` respectively. 

To test:
- add a shortcode :D 
`[ghactivity_average_label_time repo="automattic/jetpack" label="&#91;Status&#93; Needs Review"]`
or just check it live: http://jeherve.wpsandbox.me/average-time-of-prs-being-labeled-with-needs-review-label/

Also, you can:
- add new Query Label Slug (via GitHub Query Records CPT) e.g. `automattic/jetpack#[Status] Needs Design Review`
- re-run `GHActivity_Schedule->record_average_label_times()` 2 times to get some data
- add a shortcode to a new page.
